scripts/israel.js

```diff
@@ -0,0 +1,10 @@
+var text="hello world,Am ";
+var fullname="Israel ";
+var word="from ";
+var company="HNGi7 ";
+var id="HNG-04195 ";
+var text1="using ";
+var language="javascript language ";
+var stage="for stage 2 task email is,";
+var email="adeoyefemi3@gmail.com";
+console.log(text+fullname+word+company+id+text1+language+stage+email);

```

Comment:
wrong output. 
1 text = it should be " Hello World, this is"
2. no spaces!!
3. company = should be "HNGi7 ID"
4 .email comes before text1 and add "and" after id

Reply:
language should be "javascript" not javascript language

Reply:
stage= "for stage 2 task"